docs/src/Sheet.doc.js

```diff
@@ -23,472 +30,445 @@ card(
+                  label="Mobile Measurement Partners (MMP)"
+                  name="source"
+                  value="mmp"
+                  onChange={() => {}}
+                  id="mmp"
+                />
+                <RadioButton
+                  label="Conversion Upload"
+                  name="source"
+                  value="conversion"
+                  onChange={() => {}}
+                  id="upload"
+                />
+                <RadioButton
+                  label="API"
+                  name="source"
+                  value="api"
+                  onChange={() => {}}
+                  id="api"
+                />
+              </Flex>
+            </Fieldset>
+          </Flex>
+          <Flex direction="column" gap={4}>
+            <Box>
+              <Text inline weight="bold">Step 3:</Text>
+              <Text inline> Set a filter</Text>
+            </Box>
+            <TextField id="users" label="Users in the past few days" placeholder="Ex. 4" onChange={() => {}}/>
+            <Checkbox label="Include past traffic data" name="traffic" id="traffic" onChange={() => {}}/>
+          </Flex>
+        </Flex>
+      </Sheet>
+    );
+  };
 
-card(
-  <Example
-    id="closeOnOutsideClickExample"
-    name="Prevent close on outside click"
-    description={`
-      Sometimes we need the user to complete some required action at a Sheet. We can increase the user focus by preventing the user closing the sheet from clicking on the backdrop (gray area) outside of the Sheet.
-      For that you can set: \`closeOnOutsideClick\` to \`false\`.
-      PS: The user will still be able to close the Sheet via the Dismiss button and the ESC key.
-    `}
-    defaultCode={`
-function CloseOnOutsideExample(props) {
-  const [shouldShow, setShouldShow] = React.useState(false);
+  const [shouldShow, setShouldShow] = React.useState(true);
   const HEADER_ZINDEX = new FixedZIndex(10);
   const sheetZIndex = new CompositeZIndex([HEADER_ZINDEX]);
 
   return (
     <React.Fragment>
       <Button
         inline
-        text="Open sheet"
+        text="View example Sheet"
         onClick={() => setShouldShow(true)}
       />
       {shouldShow && (
         <Layer zIndex={sheetZIndex}>
-          <Sheet
-            accessibilityDismissButtonLabel="Dismiss"
-            accessibilitySheetLabel="Example sheet to demonstrate preventing close on outside click"
-            closeOnOutsideClick={false}
-            heading="Sheet that can't be closed by clicking outside"
-            onDismiss={() => setShouldShow(false)}
-            size="lg"
-          >
-            <Text>Click on the dismiss button or press the ESC key to close the sheet.</Text>
-          </Sheet>
+          <SheetWithAccessibilityLabels onDismiss={() => setShouldShow(false)} />
         </Layer>
       )}
     </React.Fragment>
   );
 }
 `}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Focus management"
```

Comment:
Should we add an example for this one? Probably not sandboxed, so we can demonstrate how focus is placed when the Sheet opens

Reply:
Once we fix the focus bug, yes :P

Reply:
http://gph.is/1E2ssrH